client/src/components/CourseView.js

```diff
@@ -23,13 +24,16 @@ const dummyPastInstances = [{ period: '2020-2021 Autumn I-II', startDate: new Da
   { period: '2019-2020 Autumn I-II', startDate: new Date(2019, 8, 9), endDate: new Date(2019, 11, 8), type: 'Lecture', id: 2 }];
 
 const CourseView = () => {
+  let navigate = useNavigate();
   let { courseCode } = useParams();
 
   return(
     <Box sx={{ mr: -4, ml: -4 }}>
       <Box sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', alignItems: 'center', mb: 4 }}>
         <Typography variant='h3' sx={{ fontWeight: 'light' }}>{courseCode + ' – ' + dummyCourseName}</Typography>
-        <Button size='large' variant='contained'>New instance</Button>
+        <Button size='large' variant='contained' onClick={() => { navigate('/fetch-instances/' + courseCode); }}>
```

Comment:
Does this kind of button allow you to e.g. middle-click it in a browser, to open the target in a new tab?

Reply:
Opening a new tab is not at the moment possible but I could look into it how to implement it. With a quick investigation I didn't find a simple fix for this yet.

Reply:
I've found something that could be of interest:
https://mui.com/material-ui/guides/routing/

Reply:
I could look into this during the next sprint since this is something that could be implemented throughout the project but I think that for now I'm going to dismiss this so that this PR could be closed and merged.